[2024-03-29T00:00:01.685+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available scheduled__2024-03-28T00:00:00+00:00 [queued]>
[2024-03-29T00:00:01.688+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available scheduled__2024-03-28T00:00:00+00:00 [queued]>
[2024-03-29T00:00:01.689+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-03-29T00:00:01.689+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-03-29T00:00:01.689+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-03-29T00:00:01.694+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-03-28 00:00:00+00:00
[2024-03-29T00:00:01.697+0000] {standard_task_runner.py:55} INFO - Started process 18633 to run task
[2024-03-29T00:00:01.699+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'scheduled__2024-03-28T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmprvy3jn6j']
[2024-03-29T00:00:01.700+0000] {standard_task_runner.py:83} INFO - Job 67: Subtask is_api_available
[2024-03-29T00:00:01.727+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available scheduled__2024-03-28T00:00:00+00:00 [running]> on host 2be32c994b3e
[2024-03-29T00:00:01.759+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-03-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-28T00:00:00+00:00
[2024-03-29T00:00:01.760+0000] {http.py:120} INFO - Poking: api/
[2024-03-29T00:00:01.767+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-03-29T00:00:01.771+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-03-29T00:00:02.532+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-03-29T00:00:02.539+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20240328T000000, start_date=20240329T000001, end_date=20240329T000002
[2024-03-29T00:00:02.574+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-03-29T00:00:02.593+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
